test/service/test_response.py

```diff
@@ -164,16 +162,16 @@ def test_key_search_files_response(self):
                             "version": "2001-03-16T05:26:40"
                         }
                     ],
-                    "processes": [
-                        {
-                            "instrumentManufacturerModel": ["green"],
-                            "libraryConstructionApproach": ["fuchsia"],
-                            "processId": ["maroon"],
-                            "processName": ["olive"],
-                            "protocol": ["olive"],
-                            "protocolId": ["green"]
-                        }
-                    ],
+                    "protocols": [
+                            {
+                                "instrumentManufacturerModel": None,
```

Comment:
This doesn't look right. It looks as if it were the result of grouped aggregation which you removed.